src/main/java/step3/model/Car.java

```diff
@@ -0,0 +1,34 @@
+package step3.model;
+
+import java.util.Random;
+
+public class Car {
+    private static final int INITIAL_COUNT = 0;
+    private static final int MOVE_MIN_NUMBER = 4;
+    private static final int RANDOM_MAX_NUMBER = 10;
+    private static final Random random = new Random();
```

Comment:
`Random` 인스턴스도 상수로 선언하셨으니, 이름을 대문자로 작성해보면 어떨까요?
```suggestion
    private static final Random RANDOM = new Random();
```